packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -0,0 +1,43 @@
+Dropdowns are containers into which you can insert elements. The dropdown containers do not function independently; they must be opened via a dropdown button, for example. The dropdown container contains items that offer the user a specific selection. These items can be customized depending on their intended use.
+
+Caution, only the dropdown container and its items are described here! For information about the element that opens it, read the article about the dropdown button or the split button.
+
+![Overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2353-2278&mode=design&t=OVHeXvLZYLkP2CzN-4)
+
+1. Dropdown Container
+2. Item
+3. Header
+4. Quick Actions
+5. Submenu Item
+6. Checked
+7. Separator
+8. Icon
+9. Label
+
+## Options
+
+- **Header:** Add an header for the dropdown container. 
+- **Quick Actions:** Add a quick action bar.
+- **Checked:** Mark selected items in the dropdown with a check mark.
+- **Submenu:** Add an submenu for a multi-level dropdown.
+- **Separator** Add a separator to visually divide items from each other 
+- **Icon:** Select, if required, a fitting icon for the item.
```

Comment:
try to make this sentence less complicated (you could align with the wording from the button or pane chapter)